test/evp_test.c

```diff
@@ -1818,8 +1818,11 @@ static int encode_test_run(EVP_TEST *t)
             goto err;
 
         EVP_EncodeInit(encode_ctx);
-        EVP_EncodeUpdate(encode_ctx, encode_out, &chunk_len,
-                         expected->input, expected->input_len);
+        if (!TEST_true(EVP_EncodeUpdate(encode_ctx, encode_out, &chunk_len,
+                                        expected->input, expected->input_len))){
+            EVP_ENCODE_CTX_free(encode_ctx);
```

Comment:
`goto err;` only, and cleanup `encode_ctx` below, see my other comment.

Reply:
ok, let's do it.